pandas/tests/arrays/floating/test_arithmetic.py

```diff
@@ -180,3 +180,43 @@ def test_cross_type_arithmetic():
     result = df.A + df.B
     expected = pd.Series([2, np.nan, np.nan], dtype="Float64")
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "source, target",
+    [
+        ([1.1, 2.2, 3.3], [-1.1, -2.2, -3.3]),
+        ([1.1, 2.2, None], [-1.1, -2.2, None]),
+        ([-1.1, 0.0, 1.1], [1.1, 0.0, -1.1]),
+    ],
+)
+def test_unary_minus_float(float_ea_dtype, source, target):
+    dtype = float_ea_dtype
```

Comment:
please add a reference to the GitHub issue here:
```
# GH38794
```

Reply:
reference added